modules/administration-guide/pages/enabling-users-to-run-multiple-workspaces-simultaneously.adoc

```diff
@@ -0,0 +1,47 @@
+:_content-type: PROCEDURE
+:navtitle: Enabling users to run multiple workspaces simultaneously
+:description: Enabling users to run multiple workspaces simultaneously
+:keywords: administration guide, number, workspaces
+
+[id="enabling-users-to-run-multiple-workspaces-simultaneously_{context}"]
+= Enabling users to run multiple workspaces simultaneously
+
+By default, a user can run only one workspace at a time. You can enable users to run multiple workspaces simultaneously.
+
+NOTE: If using the default `common` (or `per-user`) storage method, users might experience problems with concurrently running workspaces if pods are distributed across nodes in the cluster. Switching to `per-workspace` storage, or using the `ephemeral` storage type, can help with this. 
```

Comment:
```suggestion
NOTE: If using the default `common`, that is or `per-user`, storage method, users might experience problems with concurrently running workspaces if pods are distributed across nodes in the cluster. Switching to `per-workspace` storage, that is by using the `ephemeral` storage type, can help with this. 
```

Reply:
I'm not sure but I think "per user" is not actually a supported option, it's just a description of how `common` works. 

So... using "that is" here seems awkward. 

How about:

```
NOTE: If using the default storage method, users might experience problems with concurrently running workspaces, if pods are distributed across nodes in a multi-node cluster. Switching from the per-user `common` storage strategy to the `per-workspace` storage strategy, or using the `ephemeral` storage type can avoid or solve those problems.
```

Reply:
`If using the default storage method and if pods are distributed across nodes in a multi-node cluster` seems harder to parse than my option, but that could just be my opinion. Perhaps run this past your language reviewer folks?

Reply:
I updated my suggestion in response. Is the suggestion okay now?